Bubble/app/reducers/Reducer.js

```diff
@@ -186,7 +188,8 @@ export default function Reducer(state = initialState, action) {
             myRooms: {
               refreshing: false,
               data: action.payload,
-            }
+            },
+            joinedRooms:  action.payload
```

Comment:
joined rooms is not the same as my rooms.
joined rooms are rooms i'm in, my rooms are rooms i created